lib/shared/drive-constraints.js

```diff
@@ -142,7 +142,31 @@ exports.isSourceDrive = (drive, image) => {
  * }
  */
 exports.isDriveLargeEnough = (drive, image) => {
-  return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'size' ], UNKNOWN_SIZE);
+  const driveSize = _.get(drive, [ 'size' ], UNKNOWN_SIZE);
+
+  if (_.get(image, [ 'size', 'final', 'estimation' ])) {
```

Comment:
If the image is `null`, then this line will resolve to `undefined`, which is casted to `false`, and eventually results in `true` in the last line of this function.